modules/core/src/shaderlib/misc/geometry.js

```diff
@@ -30,9 +30,15 @@ struct VertexGeometry {
 `;
 
 const fs = `
+#define SMOOTH_EDGE_RADIUS 0.5
```

Comment:
Any chance the user will want to control this (maybe to create a fade out effect)?

Reply:
I'd keep it under the wraps for the current release cycle (i.e. a visual artifacts fix vs. a new feature)